### DISCUSSION TITLE
I need someone to check if this chrome extension is working, that I made.

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#726] [(A) Anything]
I made a respitory with no README.md to make sure it doesn't get messed up. 
Please someone check the extension, because I can't install it. I don't have dev mode.
https://github.com/hellonearth311/SimpleExtension

**Comment by ghost** [comment]
I could do it but what does the extension do?

**Reply by hellonearth311** [reply]
it measures the website speed of a page.
Also, please download the latest release ZIP file from this [respitory.](https://github.com/GTMetrixPlugin-Extension/GTMetrixExtension)
